mods/d2k/maps/atreides-02a/map.yaml

```diff
@@ -205,9 +205,9 @@ Rules:
 	carryall.reinforce:
 		Cargo:
 			MaxWeight: 10
-	construction_yard:
-		Production:
-			Produces: Building
+	upgrade.conyard:
+		Buildable:
+			Prerequisites: ~disabled
```

Comment:
This won't give us the desired results, as you can still buy the barracks upgrade which has no effect.
You probably want to use
```
Player:
	-Production:
```
instead (on the other missions except 03a too).